mkdocs.yml

```
@@ -85,13 +85,44 @@ nav:
         - './microservices/support/scheduler/Ch-Scheduler.md'
         - './microservices/support/eKuiper/Ch-eKuiper.md'
       - Devices Services:
+        - Supported Device Services:
+          - './microservices/device/Ch-DeviceServiceList.md'
+          - Device Onvif:
+            - './microservices/device/ONVIF/General.md'
```

Please make the sub-folder match the full name
```suggestion
            - './microservices/device/supported/device-onvif-camera/General.md'
```